package.json

```diff
@@ -54,5 +54,9 @@
         12
       ]
     }
-  }
+  },
+  "files": [
```

Comment:
As a further observation, @ljharb, `tape` is a dev dependency. If a user goes into the `deep-equal` folder in their `node_modules` and tries to run the tests, they will fail because `tape` isn't installed.